src/utils.rs

```diff
@@ -4,19 +4,6 @@
 
 use ffi;
 use translate::*;
-use UserDirectory;
-
-pub fn get_application_name() -> Option<String> {
```

Comment:
Only https://github.com/gtk-rs/gir/issues/368 currently, nothing else I'm aware of at this point. We'll see. What do you target as release date?

Other API breaking change was mine to add a lifetime parameter to FromValue